testutils/factories/draftReferral.ts

```diff
@@ -104,23 +104,23 @@ class DraftReferralFactory extends Factory<DraftReferral> {
     })
   }
 
-  filledFormUpToCompletionDate(serviceCategories: ServiceCategory[] = [serviceCategoryFactory.build()]) {
+  filledFormUpToEnforceableDays(serviceCategories: ServiceCategory[] = [serviceCategoryFactory.build()]) {
```

Comment:
Did you change the order here to have the factory methods in the same order as the order of questions in the referral form, or is there another reason? If it's just the former, I'd say maybe no need next time we have to change the order of stuff in the referral form.

Reply:
yes that is correct. I realise that it's unnecessary, but I think it would help a bit in terms of reading the code as this section of the code can be a bit confusing.